engine/execution/state/state.go

```diff
@@ -61,9 +61,6 @@ type ReadOnlyExecutionState interface {
 type ExecutionState interface {
 	ReadOnlyExecutionState
 
-	// CommitDelta commits a register delta and returns the new state commitment.
-	CommitDelta(context.Context, delta.Delta, flow.StateCommitment) (flow.StateCommitment, error)
-
```

Comment:
If we are not using commit delta at execution state we should probably port the method over to committer.